feat(blog): add gatsby preview beta blog post (#12764)
gatsbyjs/gatsby: docs/blog/2019-03-22-introducing-gatsby-preview-beta/index.md

---
title: Introducing Gatsby Preview Beta
date: 2019-03-22
author: Marisa Morby
excerpt: "Introducing the Gatsby Preview Beta: learn more and sign up today for early access to our first commerical product."
tags:
  - preview
  - cloud
  - commercial
---

## What we’ve been up to

> Back in July we were excited to send out our [alpha version of Gatsby Preview](https://www.gatsbyjs.org/blog/2018-07-17-announcing-gatsby-preview/). The team has been working hard getting your feedback and incorporating it into Gatsby Preview beta.

Reviewer: I think this would benefit from a stronger, more active lead sentence (`excited` also fits great in the next paragraph):
```suggestion
Back in July we sent out our [alpha version of Gatsby Preview](https://www.gatsbyjs.org/blog/2018-07-17-announcing-gatsby-preview/). The team has been working hard getting your feedback and incorporating it into Gatsby Preview beta.
```

> We’re excited to let you know that this week, Gatsby Preview beta will be officially opening!

Reviewer: What does "officially opening" mean in this context? When I got to the end of the post and it talked about a closed beta, I wasn't sure.

_In beta, we’ll be supporting Gatsby sites that are set up with Contentful._

## What Gatsby Preview can do for you and your team

In a traditional CMS setup, content creators and editors can preview their changes before they go live. Without Gatsby Preview, teams using Gatsby needed to set up their own preview infrastructure or do without. This was a problem because content teams need to see what changes they’re making as it’s happening. And assuming teams had time to create their own preview infrastructure, it was still a time-consuming process. We wanted to solve this for teams using Gatsby.

With Gatsby Preview beta, you can connect your GitHub repositories and Contentful CMS to our service, and get a live URL where content creators can see their changes in context before going live. Want to check on your new article title, or look at the changes you made to your post? Gatsby Preview lets you see it!

## What’s new with Gatsby Preview beta

> For the beta version of Gatsby Preview, we focused on adding a UI over the existing functionality, made the initial set-up less manual, and added ways for your team to get involved. In beta, we’ll only be supporting Gatsby sites managed through the Contentful CMS with support for other CMS’s coming soon.

Reviewer: ```suggestion
For the beta version of Gatsby Preview, we focused on adding a UI over the existing functionality, made the initial set-up less manual, and added ways for your team to get involved. In beta, we’ll only be supporting Gatsby sites managed through the Contentful CMS with support for other CMS platforms coming soon.
```
A suggestion since CMS's is a little awkward.

### Easier client management

With agencies in mind, we’re making it easier for your clients to preview the site you’ve been working on, too. Just send them the preview URL, or add them to your GitHub repository and invite them as a contributor to Gatsby Preview so they can view their site using the Gatsby Preview dashboard. Every time a content change is typed into Contentful, your client can see the change through their Gatsby Preview URL!

For agencies that have several clients on Contentful, or if you’re managing multiple repos, the Gatsby Preview dashboard lets you set up, view, and manage multiple sites in one place.

### Easier troubleshooting

We know it’s particularly frustrating if something fails to build but there’s no way to troubleshoot it. With beta, you’ll be able to see the status of your current preview instance, view or copy your build logs, and trigger a rebuild of your preview instance.

See how you can troubleshoot and add or remove contributors:

> <iframe width="560" height="315" src="https://www.youtube.com/embed/kN5vrRVD_6Q" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Reviewer: ```suggestion
<iframe width="560" height="315" title="Video: Gatsby Preview Settings Options" src="https://www.youtube.com/embed/kN5vrRVD_6Q" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

## How to sign up

We’re currently running a closed beta for Gatsby Preview. [Sign up here](https://www.gatsbyjs.com/preview/) to get added to the waitlist.

## What’s coming next

The initial invites for the closed beta have just gone out! We’ll be talking directly with beta users so we can continue learning and make the experience even better.

Curious to see Gatsby Preview in action? Watch our short demo to see how it works:

> <iframe width="560" height="315" src="https://www.youtube.com/embed/RrQi6W_8SVQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Reviewer: ```suggestion
<iframe width="560" height="315" title="Video: Gatsby Preview and Contentful demo" src="https://www.youtube.com/embed/RrQi6W_8SVQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
